### PR TITLE
Test on PHP 7.3 and 7.4

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,13 +1,5 @@
 language: php
-php: [ 7.1, 7.2, nightly ]
-sudo: false
-
-env:
-
-matrix:
-  fast_finish: true
```
Comment (mvriel) [inline .travis.yml]: Hey Graham, why did you remove the fast_finish? When the matrix is executing I would expect this to be on so that you don't have to wait for all steps

File: .travis.yml [hunk 2]
```diff
@@ -1,13 +1,5 @@
 language: php
-php: [ 7.1, 7.2, nightly ]
-sudo: false
```
Comment (mvriel) [inline .travis.yml]: Thanks, I hadn't noticed we used this outdated property

File: .travis.yml (hunk 1 above)
Comment (GrahamCampbell) [reply .travis.yml]: I can add it back if you like?

Comment (mvriel) [reply .travis.yml]: Yes please, then I can merge this PR as soon as it goes green :)

Comment (GrahamCampbell) [reply .travis.yml]: btw, fast finish was not in the travis file at https://github.com/phpDocumentor/ReflectionDocBlock/pull/189

Comment (mvriel) [reply .travis.yml]: Ay, that is an omission there :S

Comment (mvriel) [reply .travis.yml]: Thanks for the quick fix, as soon as the build goes green I will merge it!
